Simple-Store.md

```diff
@@ -0,0 +1,87 @@
+    - [Publishing a create topic to create the workorders](#publishing-a-create-topic-to-create-the-workorders)
+  - [Option to switch between array store and persistence](#option-to-switch-between-array-store-and-persistence)
+  - [Removing data from the data browser in the deployed raincatcher-demo-cloud and raincatcher-demo-auth applications](#removing-data-from-the-data-browser-in-the-deployed-raincatcher-demo-cloud-and-raincatcher-demo-auth-applications)
+  - [Resetting a persistent storage raincatcher project](#resetting-a-persistent-storage-raincatcher-project)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Use Simple Perstistence Store
+
+# Overview
+- This is a new module raincatcher-store-simple that will
+    - use current functionality in the mediator ArrayStore 
+    - use the $fh.db cloud API to store data in a mongo database
+-  The cloud applications can specify a `WFM_USE_MEMORY_STORE` environment variable 
+    - allow users to control whether the memory store or the MongoDB store is used.
+
+## Use Simple Perstistence Store for workorders in Cloud App
+
+### Create Simple Store in your Cloud App
+```js
+const SimpleStore = require('fh-wfm-simple-store');
+const simpleStore = new SimpleStore('workorders');
+simpleStore.init({/* init with data */}).then(function() {
+  // store is available for use
+});
+```
+### Registering CRUD topics for the workorders
+```js
+simpleStore.listen('cloud:', mediator);
+```
+
+### Create list of of workorders to be saved into Simple Store 
+```js
+var workorders = [
+    { id: "rkX1fdSH", workflowId: 'SyVXyMuSr', assignee: 'rkX1fdSH', type: 'Job Order', title: 'Footpath in disrepair', status: 'New', startTimestamp: '2015-10-22T14:00:00Z', address: '1795 Davie St, Vancouver, BC V6G 2M9', location: [49.287227, -123.141489], summary: 'Please remove damaged kerb and SUPPLY AND FIX 1X DROP KERB CENTRE BN 125 X 150 cart away from site outside number 3.'},
+  ...
+    { id: "BkuXajsIB", workflowId: 'SyVXyMuSr', assignee: null, type: 'Job Order', title: 'Sink in disrepair', status: 'New', startTimestamp: '2015-10-22T07:00:00Z', address: '3820 Oak St, Vancouver, BC V6H 2M5', location: [49.251362, -123.127070], summary: 'Please remove damaged kerb and SUPPLY AND FIX 1X DROP KERB CENTRE BN 125 X 150 cart away from site outside number 3.'};
+];
+```
+
+### Reset the workorder dates to today to allow viewing workorders in the scheduler 
+```js
+  var today = new Date();
+  var tomorrow = new Date();
+  tomorrow.setDate(today.getDate() + 1);
+  workorders.forEach(function(workorder, index) {
+    var date = new Date(workorder.startTimestamp);
+    var hours = date.getHours();
+    var newDate = index < workorders.length * 2 / 3 ? today : tomorrow;
+    newDate.setHours(hours);
+    workorder.startTimestamp = newDate.getTime();
+  });
+```
+
+### Publishing a create topic to create the workorders
+```js
```

Comment:
It would be good to have a sentence that identifies that the `wfm:cloud:workorder:create` topic is picked up by the workorder module, which in turn publishes the data store topic.

See (https://github.com/feedhenry-raincatcher/raincatcher-message/pull/7/files#diff-c945a46d13b34fcaff544d966cffcabaR14) for an example in the fh-wfm-message module